app/controllers/api/v1/cities_controller.rb

```diff
@@ -0,0 +1,20 @@
+module Api
+  module V1
+    class CitiesController < ApplicationController
+      def index
+        state = State.includes(:cities).find(params[:state_id])
+        cities = state.cities.map do |city|
+          {
+            id: city.id,
+            city_name: city.name,
+            state_name: state.name,
+            state_abbrev: state.abbrev
+          }
+        end
+        render json: cities, status: :ok
+      rescue ActiveRecord::RecordNotFound
+        render json: { error: 'State not found' }, status: :not_found
+      end
+    end
+  end
+end
```

Comment:
Good job implementing your controller.  It is correct to render json because that will mate easier for the front-end to get and display this data.